### PR TITLE
Clean up some existing varlink endpoints

### PR DESCRIPTION
**Pull request by baude**
Going through and adding options (like tls-verify, signature option, etc)
to some varlink endpoints (like push/pull) many of which had not been
updated since their original authoring.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: libpod/pod.go [hunk 1]
```diff
@@ -292,3 +294,36 @@ func (p *Pod) GetPodStats(previousContainerStats map[string]*ContainerStats) (ma
 	}
 	return newContainerStats, nil
 }
+
+// CreatePortBindings iterates ports mappings and exposed ports into a format CNI understands
+func CreatePortBindings(ports []string) ([]ocicni.PortMapping, error) {
```
Comment (mheon) [inline libpod/pod.go]: Why'd this move? Can it go int cmd/podman/shared?

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM other than @mheon's question.  Tests aren't hip atm.

**Comment by cevich** [comment]
ya, job is showing as 'triggered' but not actually doing anything: https://cirrus-ci.com/build/4570116530372608

Some other jobs are running fine though, and I don't see any google problems.  It looks like there was a temporary hiccup in Cirrus CI, though can't say for sure b/c I've never seen one until now :smile: I'll follow up with their support.

I bet if @baude re-pushes the Cirrus tests will start going.

File: libpod/pod.go (hunk 1 above)
Comment (baude) [reply libpod/pod.go]: it moved for varlink but yes, shared is the right place for it

**Comment by baude** [comment]
fixed up now

**Comment by baude** [comment]
can someone merge this please

**Pull request merged**